typescript/src/Common.ts

```diff
@@ -67,6 +67,24 @@ function prepareIt(): void {
             }
         });
     }
+
+    $("input[required]").bind("invalid", (e) => {
```

Comment:
The [`bind` method is deprecated](https://api.jquery.com/bind/). Should use `on` instead.